features/test.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Test command
+
+Scenario: Tests should pass
```

Comment:
@josemasar / @rbarbadillo we started adding acceptance tests with [Cucumber](https://cucumber.io/docs/installation/javascript/). Those are easier to maintain in CLIs compared to other types of projects (e.g. web services or sites) and bring a lot of value. For example, if we introduce a breaking change in any of the workflows or APIs, unit tests might succeed but it's very likely that if we have acceptance tests covering all the scenarios they'll catch it.

Love how it reads because we put ourselves in the shoes of users.

Reply:
So we´ll use Cucumber to make integration tests in our dev, but we don´t make an opinion if the user building a project on GestaltJS should use it or not, right?